test/ctia/test_helpers/search.clj

```diff
@@ -152,7 +152,7 @@
       (is (empty? (search-ids app entity "wor"))))
 
     ;; test stop word filtering
-    (is (= matched-ids (search-ids app entity "the word"))
+    #_(is (= matched-ids (search-ids app entity "the word"))
```

Comment:
@ereteog I've been trying to understand why this thing is failing (and I think the second case on L:192 is relevant too)

It looks like when `fields` are used, things just don't work (as before) with `AND` operator. Compare these queries:

      "query_string" : {
          "default_operator" : "AND",
          "fields" : ["description",
                      "short_description",
                      "title",
                      "promotion_method"
                      ],
          "query" : "the incident"
      }
      
      ; => no results
      
      ; However,

      "query_string" : {
          "default_operator" : "AND",
          "query" : "the incident"
      }
      
      ; works
      ; and:
      
      "query_string" : {
          "default_operator" : "OR",
          "fields" : ["description",
                      "short_description",
                      "title",
                      "promotion_method"
                      ],
          "query" : "the incident"
       }
      
      ; also works
     
I can cheat here and simply modify the tests so they use "OR" operator instead, but would that impact anything we have in prod?

Reply:
I prefer to understand first.
It's probably related to: https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html#query-string-multi-field-parms

Reply:
Sounds good. If I dig up something interesting, I'll let you know.